# Header header
float32 time
float32 xPos
float32 yPos
float32 yaw
float32 xVel
float32 yVel
float32 zAngVel
float64 cmd_xVel
float64 cmd_zAngVel